### ISSUE TITLE
Exception thrown while invoking open

### ISSUE THREAD

**Issue by yamill** [chriselly/react-native-mediaplayer#1]
I'm getting this error when trying to play a file.

``` javascript

Exception thrown while invoking open on target RNMediaPlayer with params (
        {
        uri = "http://out.wpv.com.s3.amazonaws.com/hls/9Nd4vN8W7r-playlist.m3u8";
    }
): Media not found: http://out.wpv.com.s3.amazonaws.com/hls/9Nd4vN8W7r-playlist.m3u8
```

![ios simulator screen shot aug 6 2015 10 17 31 am](https://cloud.githubusercontent.com/assets/755943/9113788/734a4506-3c24-11e5-9a20-1469ba905abc.png)

**Comment by chriselly** [comment]
@yamill Yeah I think this needs to be done still to handle streaming from a URL. The first implementation I only got as far as implementing local files.

**Comment by yamill** [comment]
Ah ok cool. Any plans for handling URL's? I can probably look into this and submit a PR.

**Comment by chriselly** [comment]
@yamill I may not be able to get to it for awhile so if you want to take a look feel free. :+1:

**Comment by yamill** [comment]
Awesome will do :+1:
